galasa-parent/dev.galasa/src/main/java/dev/galasa/api/run/Run.java

```diff
@@ -114,12 +113,22 @@ public String getObr() {
         return obr;
     }
 
-    public Boolean getLocal() {
-        return local;
+    public boolean getLocal() {
+    // To be depracated, replaced by isLocalRun() which follows naming conventions
```

Comment:
To deprecate things, you add @deprecated annotation...
And add it into the javadoc for the method.